src/Object/Post.php

```diff
@@ -170,9 +168,9 @@ public function getTemplateData($conv_responses, $thread_level = 1)
 
 		$filer = (($conv->getProfileOwner() == local_user()) ? t("save to folder") : false);
 
-		$diff_author    = ((link_compare($item['url'], $item['author-link'])) ? false : true);
-		$profile_name   = htmlentities(((strlen($item['author-name'])) && $diff_author) ? $item['author-name'] : $item['name']);
-		if ($item['author-link'] && (! $item['author-name'])) {
+		$diff_author = ((link_compare($item['url'], $item['author-link'])) ? false : true);
```

Comment:
Couldn't you directly return ```!link_compare($item['url'], $item['author-link']);```?

Reply:
Oh yeah, definitely. I didn't concentrate on language clarity but it's definitely doable.